packages/hint/.hintrc

```diff
@@ -40,6 +41,8 @@
         "strict-transport-security": "warning",
         "stylesheet-limits": "warning",
         "validate-set-cookie-header": "warning",
-        "x-content-type-options": "warning"
-    }
+        "x-content-type-options": "warning",
+        "validate-summary": "warning"
+    },
+    "extends": ["development"]
```

Comment:
I understand why you added `validate-summary`, but any particular reason for the other changes in this file?

Reply:
No, that was done earlier because I was checking some other things out. Removing the unnecessary changes.